modules/@apostrophecms/doc-type/index.js

```diff
@@ -421,7 +421,10 @@ module.exports = {
           label: 'apostrophe:shareDraft',
           modal: 'AposModalShareDraft',
           manuallyPublished: true,
-          hasUrl: true
+          hasUrl: true,
+          if: {
+            aposIsTemplate: { $ne: true }
```

Comment:
could this `if` be added via an override of `addContextMenu` that resides in the template library module?

I see that there are other references to templates in the core, maybe we should move them to the module, if not too complicated?

![image](https://github.com/apostrophecms/apostrophe/assets/8301962/3c127768-1a21-4fd7-8036-af47bdfb1a2f)